Probesys/bileto
# new: Allow to import labels

## Related issue(s)



https://github.com/Probesys/bileto/issues/283

## How to test manually



1. create an archive containing labels and a ticket associated to at least one label (sorry, too long to provide a working archive)
2. run the command `./docker/bin/console app:data:import /path/to/archive.zip`
3. verify that the label and the ticket are correctly imported

## Checklist
